packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
```

Comment:
For this filter to work, you will need to tag your models with `py-feat`. This will make them listed in https://huggingface.co/models?other=py-feat (0 models at the moment).

To tag a model, you must add an attribute to the model card metadata. It must be either `library_name` or under the `tags` list. I opened https://huggingface.co/py-feat/retinaface/discussions/1 as an example for you to reproduce on other models.